client/app/scripts/superdesk/editor/editor.js

```diff
@@ -13,6 +13,10 @@
 
 var TEXT_TYPE = 3;
 
+function escapeRegExp(string) {
+    return string.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
```

Comment:
Is this something you found somewhere or did you write it by yourself? Escaping can be tricky, thus it is probably best to use an existing well-tested expression.